packages/webapp/src/util/index.js

```diff
@@ -131,3 +132,13 @@ export const getObjectInnerValues = (data) => {
     return { ...reduced, [k]: data[k] };
   }, {});
 };
+
+export const truncateText = (text, length) => {
+  if (text) {
+    if (text.length > length) {
+      return text.slice(0, 22) + '...';
```

Comment:
You forgot to use `length` here.

Reply:
this works as intended, i'm not sure i understand what you mean